### ISSUE TITLE
loginCustomerWithCredentials suddenly failing

### ISSUE THREAD

**Issue by olivej** [Shopify/mobile-buy-sdk-ios#815]
I am using v2 of this SDK, have been for past year in a published app on the iTunes App Store.  Sometime in the past couple of days, calls to "loginCustomerWithCredentials" are failing with the following error :  

`Error = Error Domain=BUYShopifyErrorDomain Code=401 "(null)" UserInfo={error=Unidentified customer}`

The user information is correct.  None of this code has been touched for months, so I'm very confused about what could be going wrong.  I know that Shopify consolidated their Mobile Apps section of the admin area into a new "Private Apps" area, but that shouldn't be causing these problems and I tested all of our tokens on Feb 5 (after the consolidation) and everything was working properly.

Any insight into what would be going on is appreciated.  I feel like it is less of an issue with this SDK and more of an issue with the endpoints that this SDK touches on the Shopify side.  

The expected behavior, and the behavior for the past year, is that the call succeeds and returns a BUYCustomer object.  Instead, the above error is occurring, for any and every customer.  This is occurring on both of my Shopify stores - my development store and my production store, each of which have their own sets of Customers, etc.  This is an iOS app using Mobile BUY SDK v2.

I have talked with Shopify support in chat and posted a topic in the Developer Forum, but thought I would post here as a last resort to see if anyone else is encountering this issue right now as well.

**Comment by philk613** [comment]
I've got someone that reached out about this in our support channel who left a forum post here: https://ecommerce.shopify.com/c/shopify-apis-and-technology/t/call-to-logincustomerwithcredentials-suddenly-failing-501091

Appears to be related.

**Comment by olivej** [comment]
Yes Sir, that was me.  :)

**Comment by olivej** [comment]
If I manually request a password reset for a customer, and it is reset, the login starts working with the new password.  All of our "old" customers still can't login.

**Comment by olivej** [comment]
So I have verified this with multiple users across dev and production in our app.  If we reset their password, and then use that new password, they can log in.  If any user tries to use their "old" password, none can log in.  Did something change in the past few days with the ways passwords are hashed?  It seems like any password stored in our stores before a few days ago are not working, but if we set new ones, then users can log in again.
